### PR TITLE
Stable merge for week 30 of 2023

### PR DESCRIPTION
**Pull request by Eeems**
### New Packages
- `tailscale-systemd` - 0.0.0-1 (#706)
  - Necessary configuration for tailscale to work on the reMarkable
- `7zip` - 22.01-1 (#699)

### Updated Packages
- `koreader` - 2023.06-1 (#696 #705)
- `plato` - 0.9.34-1 (#669)
- `erode` `fret` `oxide` `rot` `tarnish` `decay` `corrupt` `anxiety` `liboxide` `notify-send` - 2.5-2 (#697)
  - Remove rm2fb runtime dependency for reMarkable 1 devices

### CODE REVIEW

**Comment by Jayy001** [comment]
Testing results

# Installing
```
reMarkable: ~/new/rmall/ opkg install *.ipk
Installing 7zip (22.01-1) to root...
Installing anxiety (2.5-2) to root...
Installing tarnish (2.5-2) to root...
Installing corrupt (2.5-2) to root...
Installing decay (2.5-2) to root...
Installing erode (2.5-2) to root...
Installing fret (2.5-2) to root...
Upgrading koreader on root from 2023.01-1 to 2023.06.1-1...
Removing obsolete file /opt/koreader/frontend/ui/widget/closebutton.lua.
Package liboxide (2.5-2) installed in root is up to date.
Installing notify-send (2.5-2) to root...
Installing oxide (2.5-2) to root...
Installing rot (2.5-2) to root...
Installing plato (0.9.34-1) to root...
Installing jq (1.6-2) to root...
Downloading https://bin.entware.net/armv7sf-k3.2/jq_1.6-2_armv7-3.2.ipk
Installing rmfm (1.4.0-3) to root...
Package rot (2.5-2) installed in root is up to date.
Installing tailscale-systemd (0.0.0-1) to root...
Package tarnish (2.5-2) installed in root is up to date.
Configuring jq.
Configuring tarnish.
Configuring fret.
Configuring anxiety.
Configuring tailscale-systemd.
Created symlink /etc/systemd/system/multi-user.target.wants/tailscaled.service → /etc/systemd/system/tailscaled.service.
Configuring 7zip.
Configuring decay.
Configuring erode.
Configuring rot.
Configuring oxide.

Run the following command(s) to use oxide as your launcher
$ systemctl enable --now tarnish

Configuring plato.

Place your media files for plato in '/home/root/plato-media'

Configuring koreader.

Run the following command(s) to use koreader as your launcher
$ systemctl enable --now koreader

Configuring corrupt.
Configuring notify-send.
Configuring rmfm.
reMarkable: ~/new/rmall/
```

# Oxide
```
reMarkable: ~/new/rmall/ systemctl enable --now tarnish
Created symlink /etc/systemd/system/launcher.service → /lib/systemd/system/tarnish.service.
Created symlink /etc/systemd/system/multi-user.target.wants/tarnish.service → /lib/systemd/system/tarnish.service.
```
Runs smoothly, wifi, lockscreen, screenshots, notifications, gestures (had some problem with trying to get it to recognise but it did get it eventually), task switcher working as intended. Had to click "Import apps" to find the apps though (didn't find it automatically on startup). Process manager also worked fine, when application killed returns correctly to oxides main menu. Can view & take screenshots without any errors.

# 7zip
```
reMarkable: ~/new/rmall/ opkg files 7zip
Package 7zip (22.01-1) is installed on root and has the following files:
/opt/bin/7zz
/opt/usr/share/licenses/7zip/License.txt
```
In PATH, But funny name - consider renaming it to `7z` instead?
```
reMarkable: ~/toinstall/ 7zz x repo.zip 

7-Zip (z) 22.01 (armt) : Copyright (c) 1999-2022 Igor Pavlov : 2022-07-15
 32-bit arm_v:7 thumb:2 locale=C UTF8=- Threads:2

Scanning the drive for archives:
1 file, 66235562 bytes (64 MiB)

Extracting archive: repo.zip
--
Path = repo.zip
Type = zip
Physical Size = 66235562

    
Would you like to replace the existing file:
  Path:     ./index.html
  Size:     81096 bytes (80 KiB)
  Modified: 2023-07-30 17:55:56
with the file from archive:
  Path:     index.html
  Size:     81096 bytes (80 KiB)
  Modified: 2023-07-26 17:41:56
? (Y)es / (N)o / (A)lways / (S)kip all / A(u)to rename all / (Q)uit? Y

Everything is Ok                              

Folders: 3
Files: 22
Size:       66290277
Compressed: 66235562
reMarkable: ~/toinstall/ ls
index.html  repo.zip    rm1         rm2         rmall

reMarkable: ~/toinstall/ 7zz a repo.zip test

7-Zip (z) 22.01 (armt) : Copyright (c) 1999-2022 Igor Pavlov : 2022-07-15
 32-bit arm_v:7 thumb:2 locale=C UTF8=- Threads:2

Open archive: repo.zip
--
Path = repo.zip
Type = zip
Physical Size = 66235562

Scanning the drive:
1 file, 0 bytes

Updating archive: repo.zip

Keep old data in archive: 3 folders, 22 files, 66290277 bytes (64 MiB)
Add new data to archive: 1 file, 0 bytes

                                           
Files read from disk: 1
Archive size: 66235682 bytes (64 MiB)
Everything is Ok

reMarkable: ~/toinstall/ 7zz x testfile.rar3.av.rar 

7-Zip (z) 22.01 (armt) : Copyright (c) 1999-2022 Igor Pavlov : 2022-07-15
 32-bit arm_v:7 thumb:2 locale=C UTF8=- Threads:2

Scanning the drive for archives:
1 file, 327 bytes (1 KiB)

Extracting archive: testfile.rar3.av.rar
--
Path = testfile.rar3.av.rar
Type = Rar
Physical Size = 327
Solid = -
Blocks = 1
Multivolume = -
Volumes = 1

Everything is Ok

Size:       12
Compressed: 327
```
Very fast. 

# KoReader
Launched via oxide, works fine. Connects to my calibe library and no crashes when reading books.

# Plato
Launched via koreader, detects and can read books fine. Applications work as intended. 

# Rot
```
reMarkable: ~/plato-media/ rot apps get runningApplications
{"Plato":"/codes/eeems/oxide1/apps/6cd104aa811a5d25bca5c90a23b22ce3","codes.eeems.corrupt":"/codes/eeems/oxide1/apps/661a3023fc785f5b8d244960d7da0db5","codes.eeems.fret":"/codes/eeems/oxide1/apps/f3999e4363cb568ea6ccc50aa4b27b23"}
```
As for decay, corrupt, tarnish, erode, fret and anxiety they all work fine as stated in the `Oxide` section.

tl;dr looks good to me 👍

**Comment by Jayy001** [comment]
Forgot to add tailscale,
```
[[0;1;32m●[[0m tailscaled.service - Tailscale node agent
     Loaded: loaded (/etc/systemd/system/tailscaled.service; enabled; vendor preset: disabled)
     Active: [[0;1;32mactive (running)[[0m since Sun 2023-07-30 18:07:45 UTC; 1h 58min ago
       Docs: https://tailscale.com/kb/
   Main PID: 4491 (tailscaled)
     Status: "Connected; REDACTED"
     CGroup: /system.slice/tailscaled.service
             └─4491 /opt/bin/tailscaled --state=/opt/var/lib/tailscale/tailscaled.state --socket=/opt/var/run/tailscale/tailscaled.sock --tun=userspace-networking --socks5-server=localhos
t:1055 --outbound-http-proxy-listen=localhost:1055
```
Works fine.

**Comment by Eeems** [comment]
Quick validation on my rM1 seems to also be fine.

**Review by bkirwi — APPROVED** [review]
Config looks sane, and @Jayy001's testing seems very thorough. :bowing_man:!!
